src/client/src/MainRoute.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
```

Comment:
Please git move these files so we don't lose history

Reply:
Man. I'm so bad at doing this.

Reply:
Usually git manages this pretty well — in this case it appears the resulting file is so different git didn't not regard it as the same.

I think we're past the point of amending this unless I soft revert and squash into a single commit.